pkg/operator/target_config_reconciler_test.go

```diff
@@ -899,6 +1081,20 @@ func TestManageConfigMap(t *testing.T) {
 					Profiles: []deschedulerv1.DeschedulerProfile{deschedulerv1.LifecycleAndUtilization, deschedulerv1.RelieveAndMigrate},
 				},
 			},
+			routes: []runtime.Object{
+				&routev1.Route{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: "openshift-monitoring",
+						Name:      "prometheus-k8s",
+					},
+					Status: routev1.RouteStatus{Ingress: []routev1.RouteIngress{
+						{
+							Host: "prometheus-k8s-openshift-monitoring.apps.example.com",
+						},
+					},
+					},
+				},
+			},
```

Comment:
Shouldn't we add a case for "RelieveAndMigrate" without the prometheus route present ?